private_gpt/ui/ui.py

```diff
@@ -9,18 +9,28 @@
 from fastapi import FastAPI
 from gradio.themes.utils.colors import slate  # type: ignore
 from llama_index.llms import ChatMessage, ChatResponse, MessageRole
+from pydantic import BaseModel
 
 from private_gpt.di import root_injector
 from private_gpt.server.chat.chat_service import ChatService, CompletionGen
-from private_gpt.server.chunks.chunks_service import ChunksService
+from private_gpt.server.chunks.chunks_service import Chunk, ChunksService
 from private_gpt.server.ingest.ingest_service import IngestService
 from private_gpt.settings.settings import settings
 from private_gpt.ui.images import logo_svg
 
 logger = logging.getLogger(__name__)
 
-
 UI_TAB_TITLE = "My Private GPT"
+SOURCES_SEPARATOR = "\n\n Sources: \n"
+
+
+class Source(BaseModel):
+    file: str
+    page: str
+    text: str
```

Comment:
Nice idea to have that dedicated object, it reduce the code dup around metadata management ✌️